packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -110,11 +116,15 @@ export const VideoEffectsPaneContent = (props: {
             backgroundImageUrl: backgroundImg.backgroundProps.url
           };
           adapter.updateSelectedVideoBackgroundEffect(replaceEffect);
+          props.setActiveVideoEffect({
+            type: 'replacement',
+            timestamp: new Date(Date.now())
+          });
           await adapter.replaceVideoBackground({ backgroundImageUrl: backgroundImg.backgroundProps.url });
         }
       }
     },
-    [adapter, selectableVideoEffects]
+    [adapter, props, selectableVideoEffects]
```

Comment:
avoid props in the dependency array, this will retrigger anytime any prop changes.
Instead pull out the props and pass in:
```tsx
const {setActiveVideoEffect} = props;
use...
..., [setActiveVideoEffect, ...];
```

Reply:
done